backend/prompt_studio/prompt_studio_output_manager_v2/migrations/0002_promptstudiooutputmanager_highlight_data.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.1 on 2024-12-05 10:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("prompt_studio_output_manager_v2", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="promptstudiooutputmanager",
+            name="highlight_data",
+            field=models.JSONField(
+                blank=True, db_comment="Field to store highlight data", null=True
+            ),
+        ),
+    ]
```

Comment:
Yes, It is required. we'll use cache or redis for temporary data but user expect highlight data even after long time since prompt studio is where they play around.